scripts/memory-threshold-list.yaml

```diff
@@ -1,3 +1,30 @@
+- scenarios:
+    - sample.bluetooth.fast_pair.locator_tag.release
+  platforms: # configuration for small memory devices
+    - nrf52dk/nrf52832
+    - nrf52833dk/nrf52833
+    - nrf54l15dk/nrf54l05/cpuapp
+  rom_increase: 1024
```

Comment:
Consider using smaller numbers for ROM (see nRF Desktop as an example)

Reply:
decreased the numbers

small targets -> 512 B increase triggers the notification
large targets -> 1024 B increase triggers the notification